spring-cloud-kubernetes-discovery/src/main/java/org/springframework/cloud/kubernetes/discovery/KubernetesDiscoveryClient.java

```diff
@@ -64,82 +51,86 @@ public String description() {
 	}
 
 	public ServiceInstance getLocalServiceInstance() {
```

Comment:
This method was removed from `DiscoveryClient`. It might be simpler to just delete it.